backend/pyproject.toml

```diff
@@ -8,7 +8,7 @@ license = "Apache-2.0"
 [tool.poetry.dependencies]
 python = "^3.10"
 fastapi = "^0.80.0"
-uvicorn = "^0.18.2"
+uvicorn = "^0.18.3"
```

Comment:
Backend change shouldn't be needed, possibly a rebase error?

Reply:
It is indeed an error, I haven`t make any changes to this file.

Reply:
You probably messed up during rebase, might want to be abit more careful there. I'll go ahead and fix this for now :)